src/main/java/nl/tudelft/scrumbledore/GUI.java

```diff
@@ -423,10 +441,22 @@ public void handle(KeyEvent keyPressed) {
 
         // Mapping the desired keys to the desired actions.
         if (keyPress.equals("LEFT")) {
+          // Logging the movement of the player to the game log.
+          Logger.log("Player moved to " + player.getPosition().getX() + ", "
+              + player.getPosition().getY());
```

Comment:
Maybe we should first discuss what exactly we want to log and when we want to log it. Logging a Player's position whenever it is updated might result in a lot of unnecessary lines in the log.